uat/src/main/java/com.aws.greengrass/FileSteps.java

```diff
@@ -95,4 +100,24 @@ private void addDataToFile(String data, Path filePath) throws IOException {
         }
     }
 
+    /**
+     * Arranges some log files with content on the /logs folder for a component
+     * to simulate a devices where logs have already bee written.
+     * @param componentName  name of the component.
+     * @throws IOException   thrown when file fails to be written.
+     */
+
+    @Then("I verify that 5 temporary rotated log files for component {word} are still available")
+    public void verifyActiveFile(String componentName) {
```

Comment:
This method name is misleading, maybe `verifyRotatedFilesAvailable`?

Reply:
resolved